### ISSUE TITLE
Parser error interpretation

### ISSUE THREAD

**Issue by juancoen** [raml-org/raml-js-parser-2#38]
This RAML:
```
#%RAML 0.8
---
title: API 1
...
title: API 2
```
Produces 3 errors in the Workbench:
<img width="532" alt="screen shot 2015-11-23 at 6 29 27 pm" src="https://cloud.githubusercontent.com/assets/1169708/11350419/3a7cb672-9210-11e5-9c9f-3e86d43a899d.png">

but the parser returns only one:
```
{
  "code": 1,
  "message": "JS-YAML: can not read an implicit mapping pair; a colon is missed at line 3, column 1",
  "path": null,
  "start": 28,
  "end": 29,
  "isWarning": false
}
```

Why are the two other errors not returned?
How should `start` and `end` be interpreted?

**Comment by sichvoge** [comment]
It would be good to actually return also the line number and column as separated properties.

**Comment by sichvoge** [comment]
Additionally, I would say that should be part of the documentation.

**Comment by KonstantinSviridov** [comment]
@sichvoge 
Ok, so we have to introduce a new parser-specific error interface, which has `line` and `column` instead of `start` and `end`.

**Comment by sichvoge** [comment]
so `start` and `end` is the `line` and `column`? that does not match what @juancoen reported

**Comment by petrochenko-pavel-a** [comment]
@KonstantinSviridov I think `line` and `column` should be in addition to start and end. 
Regards,
Pavel

**Comment by KonstantinSviridov** [comment]
@sichvoge 
The report of @juancoen is absolutely correct. My suggestion was to rename (start,end)=>(line,column) (assuming the implementation will be fixed). But let's do as @petrochenko-pavel-a suggests.

**Comment by juancoen** [comment]
:+1:

**Comment by blakeembrey** [comment]
Just note that `start` and `end` is a range, whereas `line` and `column` is not. The same implementation using line and column should probably be `[[line, column], [line, column]]`. Or provide a high level utility for converting `start` and `end` to the column. I don't think the words "line" or "column" should be part of the message since they are visual details left to the implementer to display correctly (either inline, with a link, etc.)

**Comment by KonstantinSviridov** [comment]
@blakeembrey , @juancoen I have created a pull request for resolving this issue:
https://github.com/mulesoft-labs/api-workbench/pull/1929.
How do you guys find it?

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.10.
